fluid/DeepASR/data_utils/data_reader.py

```diff
@@ -99,12 +111,42 @@ def generate_sample_info_list(self):
                 label_start = int(label_desc_split[2])
                 label_size = int(label_desc_split[3])
                 label_frame_num = int(label_desc_split[4])
-
-                sample_info_list.append(
-                    SampleInfo(feature_bin_path, feature_start, feature_size,
-                               feature_frame_num, feature_dim, label_bin_path,
-                               label_start, label_size, label_frame_num))
-
+                assert feature_frame_num == label_frame_num
+
+                if self._split_sentence_threshold == -1 or self._split_perturb == -1 or self._split_sub_sentence_len == -1 or self._split_sentence_threshold >= feature_frame_num:
```

Comment:
Please keep one line under the 80-character limit, which is also suitable for comments